agr_indexer/src/main/java/org/alliancegenome/indexer/indexers/curation/DiseaseAnnotationCurationIndexer.java

```diff
@@ -123,6 +116,17 @@ private List<GeneDiseaseAnnotationDocument> createGeneDiseaseAnnotationDocuments
 		return ret;
 	}
 
+	private static Map<String, Set<String>> closureMap = null;
+
+	// Get Closure for disease ontology
+	private static Map<String, Set<String>> getClosure() {
+		if (closureMap == null) {
+			DiseaseRepository diseaseRepository = new DiseaseRepository();
```

Comment:
This DiseaseRepository should be opened as part of the main class and not on each call to getClosure.

Reply:
This method is only executed once until the map is initialized. Moved it anyway.